<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCommandDialog</class>
 <widget class="QWidget" name="AddCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加指令</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,1,0">
   <item>
    <widget class="QLabel" name="label_commandName">
     <property name="text">
      <string>指令名</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_oneCommand" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_command">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0x</string>
        </property>
        <property name="placeholderText">
         <string>2字节</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QSpinBox" name="spinBox">
        <property name="maximum">
         <number>999999</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_id">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0x</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_data">
        <property name="text">
         <string>0x00</string>
        </property>
        <property name="placeholderText">
         <string>4字节</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_14">
        <property name="text">
         <string>指令</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_commandSyn">
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>0x</string>
        </property>
        <property name="placeholderText">
         <string>2字节</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>延时(ms)</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_16">
        <property name="text">
         <string>数据</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>ID</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_15">
        <property name="text">
         <string>指令序号</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget">
     <column>
      <property name="text">
       <string>ID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>指令</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>指令序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>数据</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>延时(ms)</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="layout_data">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_send">
       <property name="text">
        <string>发送指令</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_ok">
       <property name="text">
        <string>添加到列表</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
